gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -8,49 +8,46 @@ import React from 'react';
 
 import { NxCheckbox } from '@sonatype/react-shared-components';
 import { NxRadio } from '@sonatype/react-shared-components';
-
-import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
-import { faCalendar } from '@fortawesome/free-solid-svg-icons';
+import { NxButton } from '@sonatype/react-shared-components';
+import { NxStatefulTextInput } from '@sonatype/react-shared-components';
 
 export default function NxFormLayoutExample() {
+  function validator(val: string) {
+    return val.length ? null : 'Must be non-empty';
+  }
+
   return (
     <form className="nx-form">
       <div className="nx-form-row">
         <div className="nx-form-group">
           <label className="nx-label">
-            <span className="nx-label__text">Horizontal layout</span>
-            <span className="nx-sub-label">
-              <NxFontAwesomeIcon icon={faCalendar}/>
-              <span>Sub-label with icon.</span>
-            </span>
-            <input type="text" className="nx-text-input"/>
+            <span className="nx-label__text">Label</span>
+            <NxStatefulTextInput/>
           </label>
         </div>
         <div className="nx-form-group">
-          <label className="nx-label">
-            <span className="nx-label__text">Second column</span>
-            <span className="nx-sub-label">This is a sub-label.</span>
```

Comment:
What if one of the labels in a form row has a sub-label, but the other doesn't?  Won't they end up being different heights? How should they align in that case?

Reply:
> What if one of the labels in a form row has a sub-label, but the other doesn't? Won't they end up being different heights? How should they align in that case?

I mocked this up: 
<img width="870" alt="Screen Shot 2020-08-17 at 2 58 07 PM" src="https://user-images.githubusercontent.com/1991443/90434507-87231f00-e09b-11ea-9034-ba1aca42869a.png">

And @dturner32 gave his thumbs up (https://sonatype.slack.com/archives/CJUF9G7T2/p1597690790017000).